src/App.tsx

```diff
@@ -1,25 +1,41 @@
-import Login from "./pages/login";
-import Profile from "./pages/profile";
+import Login from "./Pages/Auth/login";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import Register from "./pages/register";
-import Classes from "./components/Page/Classes";
-import Revision from "./components/Page/Revision";
-import Teachers from "./components/Page/Teachers";
-import ShoppingCart from "./components/Page/ShoppingCart";
-import RevitionLimit from "./components/Page/RevitionLimit";
+
+import Layout from "./Layouts/Layout";
+import Register from "./Pages/Auth/register";
+import OTPVerify from "./Pages/Auth/OTPVerify";
+import Teachers from "./Pages/Teachers/Teachers";
+import Teacher from "./Pages/Teacher/Teacher";
+import Course from "./Pages/Course/Course";
+import Shopping from "./Pages/Shopping/Shopping";
+import ProfileUpdate from "./Pages/Auth/ProfileUpdate";
+import MyClasses from "./Pages/MyClasses/MyClasses";
+import MyClass from "./Pages/MyClass/MyClass";
+import MyLesson from "./Pages/MyLesson/MyLesson";
+import Profile from "./Pages/Auth/Profile";
 
 function App() {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/classes" element={<Classes />} />
-        <Route path="/revision" element={<Revision />} />
-        <Route path="/teachers" element={<Teachers />} />
-        <Route path="/shoppingCart" element={<ShoppingCart />} />
-        <Route path="/revisionLimit" element={<RevitionLimit />} />
-        <Route path="/register" element={<Register />} />
-        <Route path="/profile" element={<Profile />} />
-        <Route path="/" element={<Login />} />
+        <Route path="/" element={<Layout />}>
+          <Route index element={<h1>Home</h1>} />
+          <Route path="teachers" element={<Teachers />} />
+          <Route path="teacher/:id" element={<Teacher />} />
+          <Route path="course/:id" element={<Course />} />
+          <Route path="classes" element={<MyClasses />} />
+          <Route path="mycourse/:id" element={<MyClass />} />
+          <Route path="mylessons/:id" element={<MyLesson />} />
+          <Route path="profile" element={<Profile />} />
+          <Route path="payment" element={<h1>Payment</h1>} />
+          <Route path="contact" element={<h1>Contact</h1>} />
+          <Route path="shopping" element={<Shopping />} />
+          <Route path="notifications" element={<h1>Notifications</h1>} />
+        </Route>
+        <Route path="login" element={<Login />} />
+        <Route path="register" element={<Register />} />
+        <Route path="otp-verify" element={<OTPVerify />} />
+        <Route path="profile-update" element={<ProfileUpdate />} />
```

Comment:
add a reroute Route path that returns to home if path is different

Reply:
done